### DISCUSSION TITLE
Icon request: icon-paddles

### DISCUSSION THREAD

**Discussion by donnamcmaster** [FortAwesome/Font-Awesome#21130] [Icon Requests]
This is for a couple of websites in an area where whitewater rafting and kayaking are popular. On the current http://www.theamericanriver.com/ site we're using separate PNG icons for rafting and kayaking. 

![theamericanriver_com](https://cloud.githubusercontent.com/assets/421360/25638249/a1b6cb3c-2f3c-11e7-88f1-1f03ff48147f.png)

On the new sites I'm building, we're using a pair of paddles to represent both. This is our icon: 
![paddles](https://cloud.githubusercontent.com/assets/421360/25638091/15fb17a6-2f3c-11e7-8d5e-2b50fe7a1ef4.png)
![n_coloma_com-paddles](https://cloud.githubusercontent.com/assets/421360/25638612/d43dd0a4-2f3d-11e7-83d9-cafe505ea05c.png)

And the only other one I found in icon font sets (Symbolicons Block): 
![symbolicons-block-paddles](https://cloud.githubusercontent.com/assets/421360/25638115/2cb91704-2f3c-11e7-8798-b82fd8fac17e.png)

Two other example uses, these by whitewater rafting outfitters: 
![logo 2x](https://cloud.githubusercontent.com/assets/421360/25638506/81e2d6ec-2f3d-11e7-9d50-1abbeeb29174.png)
![whitewater-excitement-paddles](https://cloud.githubusercontent.com/assets/421360/25638516/8646101e-2f3d-11e7-84da-73ddd044cdf7.png)

This is related to but different from https://github.com/FortAwesome/Font-Awesome/issues/10772 (fa-raft or fa-kayak); would love to have all of them but paddles have wider use. 

Thanks for your consideration!
Donna

**Comment by donnamcmaster** [comment]
This is the Carlisle paddle which is the most ubiquitous for whitewater rafting these days. 
![epa_cegui_enlarge](https://cloud.githubusercontent.com/assets/421360/25638778/6075614a-2f3e-11e7-880c-3c9448447b33.jpg)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Added in 7.2: https://fontawesome.com/icons/oars?f=classic&s=solid
